### ISSUE TITLE
Require a note for a default.ini in docs

### ISSUE THREAD

**Issue by scriptjs** [apache/couchdb-docker#97]
I wrote a small docker-init.sh to initialize couch:
```
echo "Enter the CouchDB password to use: "
read -sr PASSWORD_INPUT

mkdir -p ~/Data/CouchDB/opt/couchdb/data
mkdir -p ~/Data/CouchDB/etc/local.d

docker run -p 5984:5984 \
  --volume ~/Data/CouchDB/opt/couchdb/data:/opt/couchdb/data \
  --volume ~/Data/CouchDB/etc/local.d:/opt/couchdb/etc/local.d \
  --env COUCHDB_USER=admin \
  --env COUCHDB_PASSWORD=$PASSWORD_INPUT \
  apache/couchdb:2.1.1
```
I received the following error:
```
chmod: cannot access '/opt/couchdb/etc/local.d/*.ini': No such file or directory
```
## Expected Behavior
No error should be emitted.

## Current Behavior
Errors as above.

## Possible Solution
I looked at the image. I  included the `10-docker-default.ini` in my `~/Data/Couchdb/etc/local.d` folder. It booted. A note and default *.ini should be referenced. 

## Steps to Reproduce (for bugs)

1. Write script.
2. Execute script to launch image.

## Context

Launch the image

## Your Environment

* Version used: 2.2.0
* Browser Name and version: FireFox 62.0b17
* Operating System and version (desktop or mobile): MacOS 10.13.6 High Sierra
* Link to your project:

**Comment by eiri** [comment]
Do you really have both `~/Data/CouchDB` and `~/Data/Couchdb` or is it a typo?

**Comment by scriptjs** [comment]
@eiri This is a typo with no impact on the issue. I have corrected it above.

**Comment by eiri** [comment]
Ok, just checking.

I think this was already fixed in apache/couchdb-docker#82 at least I can't reproduce it with your setup for image 2.2.0. Give it a try?

By the way, environment you specified doesn't match image used in your script, so I'm assuming you really meant 2.1.1.

**Comment by scriptjs** [comment]
Ah yes. I missed that I had not upgraded to 2.2.0. Verified all is good. Many thanks and apologies for the noise since it was fixed previously :)
